File: repository-remote/src/main/java/com/andyanika/translator/repository/remote/YandexTranslationResponse.java
package com.andyanika.translator.repository.remote;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

class YandexTranslationResponse {
    @SerializedName("code")
    int code;

    @SerializedName("lang")
    String languageDirection;

    @SerializedName("text")
    ArrayList<String> translatedText;
}

File: repository-remote/src/main/java/com/andyanika/translator/repository/remote/yandex/YandexTranslationResponse.kt
package com.andyanika.translator.repository.remote.yandex

import com.google.gson.annotations.SerializedName
import java.util.*

internal class YandexTranslationResponse {
    @SerializedName("code")
    var code = 0

    @SerializedName("lang")
    var languageDirection: String? = null

    @SerializedName("text")
    var translatedText: ArrayList<String>? = null
}
